README.md

```diff
@@ -92,7 +92,11 @@ yarn jest
 
 ### Building a Docker image
 
-There are two different ways to build a Grafana docker image. If your machine is setup for Grafana development and you run linux/amd64 you can build just the image. Otherwise, there is the option to build Grafana completely within Docker.
+There are two different ways to build a Grafana docker image. If your machine is setup for Grafana development and you
+run linux/amd64 you can build just the image. Otherwise, there is the option to build Grafana completely within Docker.
+
+If you had setup your environment for Grafana enterprise the docker build won't work due to symlinks. You need to remove
```

Comment:
Any way we can move these comments to the enterprise repo? I want to have a little enterprise related info in this repo as possible so that its clear for users that this repo is 100% apache2